### PR TITLE
[CentOS 7] Homologize to make rpm for centos7 

### PR DESCRIPTION
**Pull request by Mnakagawa**

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -1,7 +1,8 @@
 SUBDIRS = server client data data/init.d data/systemd
 ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
 
-EXTRA_DIST = autogen.sh README.md hatohol.spec
+EXTRA_DIST = autogen.sh README.md hatohol.spec \
+			 COPYING.GPLv3 COPYING.LGPLv3
```
Comment (bynnchapu) [inline Makefile.am]: This indent is out of alignment.

File: COPYING [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Copyright (C) 2013-2014 Project Hatohol
```
Comment (kz0817) [inline COPYING]: This year, AD 2015.

File: Makefile.am (hunk 1 above)
Comment (kz0817) [inline Makefile.am]: Why is COPYING.GPLv3 necessary ?
All source files are distributed under LGPLv3, aren't they ?

Comment (Mnakagawa) [inline Makefile.am]: How many I should use tab?
If project-hatohol decided coding rules, please tell me it.

**Comment by Mnakagawa** [comment]
I create a new pull request. Close this pullrequest.

**Pull request closed without merging**